src/test/java/org/folio/services/budget/BudgetExpenseClassTotalsServiceTest.java

```diff
@@ -112,7 +120,7 @@ void getExpenseClassTotalsComplexPositiveTest(VertxTestContext vertxTestContext)
     Transaction credit2 = buildTransaction(5.56d, Transaction.TransactionType.CREDIT, expenseClass2.getId());
     Transaction payment2 = buildTransaction(15d, Transaction.TransactionType.PAYMENT, null);
 
-    budget.withExpenditures(15d).withOverExpended(1d); // 15 + 1 = 11 + 15 - 4.44 - 5.56
+    budget.withExpenditures(15d).withOverExpended(1d).withCredits(20d);
```

Comment:
Based on the transactions above, this should be:
    `budget.withExpenditures(26d).withOverExpended(1d).withCredits(10d);`
(also the test was wrong before, it should have been `.withExpenditures(16d)` )

Reply:
used your suggession, thanks